GCI-116: create Manage Apps page with AngularJS

heidid opened the pull request:
PR for [this issue](https://issues.openmrs.org/browse/GCI-116). I recreated the Manage Apps page with AngularJS (see manage.html and main.js). It isn't integrated with the jsp templates, though. All of the rest calls to the API are functional; I was able to successfully display/upload/delete app packages.

namratanehete commented:
@enhdless very good job :) But you forgot to add ng-app tag :) So please add html tag with ng-app. Please add all css and js file imports in head tag and other code in body tag. I am uploading the screenshot in JIRA, you may need to make some changes in css. But I think thats ok. Icons are not displayed properly and after clicking on app, apps are not opening. 
But you can add html, head and body tags and submit the task. Please squash the commit and submit PR again :+1:

heidid commented:
I added the ng-app directive and doctype/html/head/body tags. The page probably looks a bit weird (for example the spacing at the top) because there is no header or footer, but if those are added then it should be fine. The apps should open now, I changed launchPath to launch_path. I am not sure about the icons, though, because they don't seem to show up for me in the current Manage Apps page either.

namratanehete commented:
@enhdless ng-app tag should have app's name. Please add some padding or margin in body tag in this html page to add some space above.

heidid committed:
create Manage Apps page with AngularJS

heidid commented:
Alright, I've added those items.

dkayiwa commented:
@sunbiz do you have any comments for this pull request? :)

dkayiwa commented:
@namratanehete do you think this is ready for merging? :)

dkayiwa commented on omod/src/main/webapp/manage.html:
```diff
@@ -0,0 +1,101 @@
+    <div class="appList" ng-controller="AppListCtrl">
+        <span ng-if="appList.length == 0"><br/>You have no apps installed.</span>
+        <div class="divTitle" ng-if="appList.length > 0">
+            <h1><span>Manage Apps:</span></h1>
```
The above text needs localization.

dkayiwa commented on omod/src/main/webapp/manage.html:
```diff
@@ -0,0 +1,101 @@
+                <thead>
+                    <tr>                     
+                        <th>Logo</th>
+                        <th id="sorting">Name</th>
```
All this text needs localization.

dkayiwa commented on omod/src/main/webapp/resources/javascript/main.js:
```diff
@@ -0,0 +1,63 @@
+    }
+
+    $scope.deleteApp = function(appName) {
+        var result = window.confirm('Do you want to delete this app?' + '\n\n' + appName);
```
The above text needs localization.

dkayiwa commented:
@enhdless this pull request has merge conflicts. Can you resolve the and create a new one?

heidid commented:
Will do, thanks for letting me know!
